### DISCUSSION TITLE
I would like my links to change according to the color of the background

### DISCUSSION THREAD

**Discussion by Stevendrawings** [community/community#21416] [Other Feature Feedback, Questions, & Ideas]
<p></p><div class="lightbox-wrapper"><a class="lightbox" href="https://aws1.discourse-cdn.com/github/original/3X/b/2/b25fe62ea1466d40de3c79b139890a047abedecd.jpeg" data-download-href="/uploads/short-url/prYs4fDLec9DNhuElmaizFKaXvL.jpeg?dl=1" title="A" rel="noopener nofollow ugc"><img src="https://user-images.githubusercontent.com/32676826/181064486-a28eacf4-b477-4a88-b998-5d9418d16f36.jpeg" alt="A" data-base62-sha1="prYs4fDLec9DNhuElmaizFKaXvL" width="690" height="225" srcset="https://aws1.discourse-cdn.com/github/optimized/3X/b/2/b25fe62ea1466d40de3c79b139890a047abedecd_2_690x225.jpeg, https://aws1.discourse-cdn.com/github/optimized/3X/b/2/b25fe62ea1466d40de3c79b139890a047abedecd_2_1035x337.jpeg 1.5x, https://aws1.discourse-cdn.com/github/optimized/3X/b/2/b25fe62ea1466d40de3c79b139890a047abedecd_2_1380x450.jpeg 2x" data-small-upload="https://aws1.discourse-cdn.com/github/optimized/3X/b/2/b25fe62ea1466d40de3c79b139890a047abedecd_2_10x10.png"><div class="meta">
<svg class="fa d-icon d-icon-far-image svg-icon" aria-hidden="true"><use href="#far-image"></use></svg><span class="filename">A</span><span class="informations">1920×628 84.7 KB</span><svg class="fa d-icon d-icon-discourse-expand svg-icon" aria-hidden="true"><use href="#discourse-expand"></use></svg>
</div></a></div>
<p>Hello, excuse me for my English, I’m from France, I’d like to reverse the color like in this image with the midnight.js plugin.</p>

**Comment by marcos-nunes-dev** [comment, marked as answer]
<p>Hello Friend! you can do this effect very simply using <code>mix-blend-mode</code>. This functionality has a good acceptance (<a href="https://caniuse.com/?search=mix-blend-mode" class="inline-onebox" rel="noopener nofollow ugc">"mix-blend-mode" | Can I use... Support tables for HTML5, CSS3, etc</a>) and is quite simple to use. I will forward an example below that was not made by me, but it is using the functionality that I am saying:</p>
<p><iframe id="cp_embed_wKqmbW" src="https://codepen.io/robinrendle/embed/preview/wKqmbW?default-tabs=css%2Cresult&amp;height=300&amp;host=https%3A%2F%2Fcodepen.io&amp;slug-hash=wKqmbW" title="A pure CSS loading bar" scrolling="no" frameborder="0" height="260" class="cp_embed_iframe" width="695" seamless="seamless" sandbox="allow-same-origin allow-scripts allow-forms allow-popups allow-popups-to-escape-sandbox allow-presentation"></iframe></p>
